src/test/java/reposense/system/CommandRunnerTest.java

```diff
@@ -58,7 +58,8 @@ public void logWithoutContentTest() {
     @Test
     public void blameRawTest() {
         try {
-            String content = CommandRunner.blameRaw(TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java");
+            String content = CommandRunner.blameRaw(
+                    TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java", null, null);
```

Comment:
instead of doing this, perhaps an overload method will make it looks cleaner?

Reply:
Hmmm, but this overloaded method would also be used for tests though?